frontend/src/Pages/AllUser/Login/ChangePassword.jsx

```
@@ -0,0 +1,100 @@
+              name="new_password"
+              value={form.new_password}
+              onChange={handleChange}
+              required
+              className="w-full px-4 py-3 text-white bg-gray-900 border border-gray-700 rounded-lg focus:outline-none focus:ring-2 focus:ring-indigo-500"
+              placeholder="Enter new password"
+            />
+          </div>
+
+          <div>
+            <label className="block mb-1 text-sm font-medium text-gray-300">Confirm New Password</label>
+            <input
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)